apps/indexer/src/eventHandlers/voting.ts

```diff
@@ -64,8 +65,10 @@ export const voteCast = async (
 export const proposalCreated = async (
```

Comment:
Type mismatch error: The schema defines `startBlock` and `endBlock` as integers, but the insertion code is passing strings. This will cause type conversion errors during database operations.